src/components/page.js

```diff
@@ -16,4 +16,14 @@ Page.propTypes = {
     children: PropTypes.element.isRequired,
 };
 
+Page.componentDidMount = () => {
```

Comment:
Can you transform your Page function into a class in order to put the `componentDidMount` as a class attribut.. ?